packages/react-dom-bindings/src/events/ReactDOMEventListener.js

```diff
@@ -57,7 +57,7 @@ import {isRootDehydrated} from 'react-reconciler/src/ReactFiberShellHydration';
 const {ReactCurrentBatchConfig} = ReactSharedInternals;
 
 // TODO: can we stop exporting these?
```

Comment:
remove this todo then or does it apply to the later exports?

Reply:
This only applied to the later exports. It's a www only thing that it reads this flag for some reason.